app/code/Bulbulatory/Recomendations/view/frontend/templates/recommendation.phtml

```diff
@@ -16,5 +16,55 @@
             <span><?php echo __('Wyślij') ?></span>
         </button>
     </div>
+</form>
 
-</form>
\ No newline at end of file
+<br></br>
+
+<?php $recommendations = $this->getRecommendationCollection();?>
+<div>
+<p><?php echo __('Ilość wysłanych poleceń: ') ?><?php echo $recommendations->getSize(); ?></p>
+</div>
+<div>
+<p><?php echo __('Ilość poterdzonych poleceń: ') ?><?php echo $this->getConfirmRecommendationCollection(); ?></p>
+</div>
+<div>
+<p><?php echo __('Zniżka: ') ?><?php echo $this->calculateDiscount() ?><?php echo __('%') ?></p>
+</div>
+
+<br></br>
+
+<?php if ($this->getPagerHtml()): ?>
+    <div class="order-products-toolbar toolbar bottom"><?php echo $this->getPagerHtml(); ?></div>
+<?php endif ?>
+
+<?php if ($recommendations && count($recommendations)): ?>
+    <div class="table-wrapper orders-history">
+        <table class="data table table-order-items history" id="my-orders-table">
+            <caption class="table-caption">Recommendation Table</caption>
+            <thead>
+            <tr>
+                <th scope="col" class="col recommendation_id">ID</th>
+                <th scope="col" class="col email">Email</th>
+                <th scope="col" class="col created_at">Created At</th>
+                <th scope="col" class="col status">Status</th>
+            </tr>
+            </thead>
+            <tbody>
+            <?php
+            foreach ($recommendations as $recommendation): ?>
+                <tr>
+                    <td data-th="ID" class="col recommendation_id"><?php echo $recommendation->getId(); ?></td>
+                    <td data-th="Email" class="col email"><?php echo $recommendation->getEmail(); ?></td>
+                    <td data-th="Created At" class="col created_at"><?php echo $recommendation->getCreatedAt(); ?></td>
+                    <td data-th="Status" class="col status"><?php echo $recommendation->getStatus(); ?></td>
```

Comment:
Teraz status wyświetla się jako 1 albo 0, dobrze by było przedstawić to bardziej przyjaźnie dla użytkownika, jako `zatwierdzony` albo `niezatwierdzony`.